std/traits.d

```diff
@@ -3279,19 +3280,7 @@ alias Identity(alias A) = A;
    Yields $(D true) if and only if $(D T) is an aggregate that defines
    a symbol called $(D name).
  */
-template hasMember(T, string name)
```

Comment:
So we're going to have `std.traits.hasMember` and `__traits(hasMember, ...)` do subtly different things? I don't think that's a good idea.